Analysis/HiggsTauTau/src/HTTSequence.cc

```diff
@@ -387,6 +429,14 @@ void HTTSequence::BuildSequence(){
   // Other flags
 
   real_tau_sample = ( (output_name.find("HToTauTau")             != output_name.npos)
+                        || (output_name.find("HTT")                   != output_name.npos)
+                        || (output_name.find("HToTautau")             != output_name.npos)
+                        || (output_name.find("T-t")                   != output_name.npos)
+                        || (output_name.find("Tbar-t")                != output_name.npos)
+                        || (output_name.find("VVTo2L2Nu")             != output_name.npos)
+                        || (output_name.find("WZTo2L2Q")              != output_name.npos)
+                        || (output_name.find("ZZTo2L2Q")              != output_name.npos)
+                        || (output_name.find("ZZTo4L")                != output_name.npos)
```

Comment:
Due to this line[*] this won't actually have an effect for 20161 data, so think we can probably remove these additions.
[*]https://github.com/ajgilbert/ICHiggsTauTau/blob/29edb847abf35fb9b11aa384dd015bee38b58bba/Analysis/HiggsTauTau/src/HTTSequence.cc#L448